package.json

```diff
@@ -123,9 +123,185 @@
     "webpack-dev-server": "4.11.1"
   },
   "engines": {
-    "node": ">=14 < 16"
+    "node": "16"
+  },
+  "build": {
```

Comment:
what is the point having the whole `electron-builder.yml` in the package json?

Reply:
one file instead of two.